run.sh

```diff
@@ -8,14 +8,14 @@ IMG=${IMG:-'quay.io/rogreen/bookmark-generator:latest'}
 OUTPUT_HTML_FILE=${OUTPUT_FILE:-'bookmark.html'}
 ADDITIONAL_CONTAINER_ENGINE_ARGS=${ADDITIONAL_CONTAINER_ENGINE_ARGS:-}
 
-${CONTAINER_ENGINE} run \
+$CONTAINER_ENGINE run \
     --rm \
     --interactive \
     --tty \
     --volume \
-        "$(pwd)":${CONTAINER_BASE_DIR}/data \
-    "${ADDITIONAL_CONTAINER_ENGINE_ARGS}" \
+    "$(pwd)":$CONTAINER_BASE_DIR/data \
+    $ADDITIONAL_CONTAINER_ENGINE_ARGS \
```

Comment:
Why is removing the quotes the solution?
Not that I'm against it, but curious

Reply:
double quote in linux always execute what's inside it, so if you try `$"ADDITIONAL_CONTAINER_ENGINE_ARGS"` you will get this error: `ADDITIONAL_CONTAINER_ENGINE_ARGS: command not found`
So whenever you want to call a variable in bash you just add $ before it's name.

Reply:
What about "$ADDITIONAL_CONTAINER_ENGINE_ARGS"
Would that also work?

Reply:
@georgettica  unfortunately no.
Please if you have confusion about the `"` , `'` and `$` in bash check the answer in this link. It does really provide a good answer for different cases. :D

Reply:
Anyways it's not a big deal..
I'll merge